src/phpDocumentor/Path.php

```diff
@@ -69,4 +72,12 @@ public static function isAbsolutePath(string $file) : bool
             )
             || parse_url($file, PHP_URL_SCHEME) !== null;
     }
+
+    public static function dirname(Path $input) : self
```

Comment:
shouldn't this be a non-static member of Path? It feels somewhat double to have a static method that accepts an instance